app/assets/javascripts/app/views/item_view.coffee

```diff
@@ -1,6 +1,63 @@
 class App.views.ItemView extends Backbone.View
-  initialize: (item) ->
-    @item = item
+  className: 'item'
+  tagName: 'li'
+  template: HandlebarsTemplates['item']
+
+  initialize: (options) ->
+    @item = options.item
+    @children = new App.collections.Items(@item.get('children'))
+    @$el.attr('id', @item.id)
```

Comment:
this isn't great practice because an item with id = '1' isn't super semantically helpful. You usually want to prepend it with something like 'item' in this instance.

If you need the raw ID value from the DOM itself, you should use the data attribute feature.
https://developer.mozilla.org/en-US/docs/Web/Guide/HTML/Using_data_attributes